AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -791,25 +832,33 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
       if ([self _shouldAbortTransitionWithID:transitionID]) {
         return;
       }
+
+      // Update display node layout
+      auto previousDisplayNodeLayout = _calculatedDisplayNodeLayout;
+      auto newDisplayNodeLayout = std::make_shared<ASDisplayNodeLayout>(
```

Comment:
Let's call this `pendingLayout` or `pendingDisplayNodeLayout`, to be consistent with the other method: https://github.com/facebook/AsyncDisplayKit/pull/2110/files#diff-230ee3845413c2a11bf13fbdb7823ddfR702